src/main/java/no/nav/familie/integrasjoner/oppgave/OppgaveService.kt

```diff
@@ -180,8 +192,14 @@ class OppgaveService constructor(
         return oppgaveRestClient.finnMapper(finnMappeRequest)
     }
 
-    companion object {
+    fun finnMapper(enhetNr: String): List<MappeDto> {
+        val finnMappeRequest = FinnMappeRequest(enhetsnr = enhetNr, limit = 1000)
```

Comment:
Hvis jeg husker rett så er det forskjell på om vi setter `tema = listOf()` og `tema = null` i søket? Ser iallefall at vi setter tom liste i søket på mottak og sak - så tror vi må ha det som tom liste for å få "treff".

Reply:
Men det er kanskje noe du har fikset i `toQueryParams` ?

Reply:
Det er noe av frovirringen jeg ønsker å fjerne med dette endepunktet. toQueryParams som vi bruker fjerner tomme lister fordi en queryString i et api ikke kan inneholde tome query-felter. Da ville du fått path?tema=enhetsnr=4488, og spørringen feiler.

Reply:
sånn var det ja 😂 Jeg bare minnes "problemer" :D